backend/vxlan/device.go

```diff
@@ -160,14 +151,9 @@ type neighbor struct {
 	IP  ip.IP4
 }
 
-func (dev *vxlanDevice) GetL2List() ([]netlink.Neigh, error) {
-	log.V(4).Infof("calling GetL2List() dev.link.Index: %d ", dev.link.Index)
-	return netlink.NeighList(dev.link.Index, syscall.AF_BRIDGE)
-}
-
-func (dev *vxlanDevice) AddL2(n neighbor) error {
-	log.V(4).Infof("calling NeighAdd: %v, %v", n.IP, n.MAC)
-	return netlink.NeighAdd(&netlink.Neigh{
+func (dev *vxlanDevice) AddFdb(n neighbor) error {
```

Comment:
I think a better name would be `AddFDB` since FDB is an acronym
See: https://github.com/golang/go/wiki/CodeReviewComments#initialisms (also applies to a bunch of other method names in this file.)